addons/account/models/partner.py

```diff
@@ -452,6 +452,7 @@ def _onchange_company_id(self):
         return {'domain': {'property_account_position_id': [('company_id', 'in', [company.id, False])]}}
 
     def can_edit_vat(self):
+        ''' One should not be able to edit `vat` if there is issued invoices. '''
```

Comment:
Maybe it's good to make possible to update till Invoice is still in Draft mode.

Reply:
Thats already the case, we search only for non draft/canceled invoices.
I could improve the docstring to mention it

Reply:
Thanks for the quick follow up.
That would be great. (and also for Sales order as well https://github.com/odoo/odoo/pull/34564/files#diff-1ead90e4fdcb815f8a141426c06a0694R33)

Reply:
Done, I changed the docstring to mention that it is only related to non draft issues :wink: